presto-hive/src/test/java/com/facebook/presto/hive/s3/TestPrestoS3FileSystem.java

```diff
@@ -65,17 +65,18 @@
 import static com.google.common.io.MoreFiles.deleteRecursively;
 import static com.google.common.io.RecursiveDeleteOption.ALLOW_INSECURE;
 import static io.airlift.testing.Assertions.assertInstanceOf;
+import static java.net.HttpURLConnection.HTTP_FORBIDDEN;
+import static java.net.HttpURLConnection.HTTP_INTERNAL_ERROR;
+import static java.net.HttpURLConnection.HTTP_NOT_FOUND;
 import static java.nio.file.Files.createTempDirectory;
 import static java.nio.file.Files.createTempFile;
-import static org.apache.http.HttpStatus.SC_FORBIDDEN;
-import static org.apache.http.HttpStatus.SC_INTERNAL_SERVER_ERROR;
-import static org.apache.http.HttpStatus.SC_NOT_FOUND;
-import static org.apache.http.HttpStatus.SC_REQUESTED_RANGE_NOT_SATISFIABLE;
 import static org.testng.Assert.assertEquals;
 import static org.testng.Assert.assertTrue;
 
 public class TestPrestoS3FileSystem
 {
+    private static final int HTTP_RANGE_NOT_SATISFIABLE = 416;
```

Comment:
airlift http-client provides this (as `io.airlift.http.client.HttpStatus#REQUEST_RANGE_NOT_SATISFIABLE`)

Reply:
I thought of this too, but that artifact has many dependencies (like Jetty HTTP client) that we don't want to add to the Hive connector. Even if we excluded them all, it seems overkill for a single well-known constant. The JAX-RS API has one as well, but same reasoning applies.

I actually filed a Guava issue about adding constants years ago.